docs/workflow/using-docker.md

```diff
@@ -59,17 +59,19 @@ Once you've chosen the image that suits your needs, you can issue `docker run` w
 docker run --rm \
```

Comment:
The Ubuntu and Alpine versions in the table above are state. We are at Ubuntu 18.04 (was 16.04 before) and Alpine 3.17 (was 3.13 before) now.

Reply:
Updated them for main images, the WASM build image I am guessing is still 16.04? @steveisok, do you happen to know?

Reply:
The wasm build image is based on azurelinux-3.0-net10.0-cross-amd64 (source: https://github.com/dotnet/dotnet-buildtools-prereqs-docker/blob/main/src/azurelinux/3.0/net10.0/webassembly/amd64/Dockerfile), so it has the same Ubuntu rootfs as azurelinux-3.0-net10.0-cross-amd64